flight/extract_gps.py

```diff
@@ -0,0 +1,94 @@
+"""Contains the extract_gps() function for extracting data out of
+the provided waypoint data JSON file for the SUAS competition."""
```

Comment:
Should split this so the quotes are on their own lines.
"""
Description
...
"""